ExternalTasks/DotNet/Program.cs

```diff
@@ -21,37 +23,35 @@ class Program
 
         static void Main(string[] args)
         {
-            RunWorker().GetAwaiter().GetResult();
+            RunWorkerNew().GetAwaiter().GetResult();

```

Comment:
Find die alte Variante besser.

Oder vllt.: 

```suggestion
            RunSampleExternalTaskWorker().GetAwaiter().GetResult();
```

Impliztes explizit machen und so. :wink: